biblio.json

```diff
@@ -1,35 +1,71 @@
 {
+	"AOM": {
+		"href": "https://wicg.github.io/aom/spec/",
+		"title": "Accessibility Object Model"
+	},
 	"BACKGROUND-FETCH": {
 		"href": "https://wicg.github.io/background-fetch/",
-		"title": "Background Fetch" 
+		"title": "Background Fetch"
+	},
+	"BUDGET-API": {
+		"href": "https://wicg.github.io/budget-api/",
+		"title": "Web Budget API"
+	},
+	"CSS-ANIMATION-WORKLET": {
+		"href": "https://wicg.github.io/animation-worklet/",
+		"title": "CSS Animation Worklet API"
+	},
+	"CSS-SCROLL-BOUNDARY-BEHAVIOR": {
+		"href": "https://wicg.github.io/scroll-boundary-behavior/",
+		"title": "CSS Scroll Boundary Behavior Module Level 1"
+	},
+	"CORS-RFC1918": {
+		"href": "https://wicg.github.io/cors-rfc1918/",
+		"title": "CORS and RFC1918"
 	},
 	"ENTRIES-API": {
 		"href": "https://wicg.github.io/entries-api/",
-		"title": "File and Directory Entries API" 
+		"title": "File and Directory Entries API"
 	},
 	"FEATURE-POLICY": {
 		"href": "https://wicg.github.io/feature-policy/",
-		"title": "Feature Policy" 
+		"title": "Feature Policy"
 	},
 	"GEOLOCATION-SENSOR": {
 		"href": "https://wicg.github.io/geolocation-sensor/",
 		"title": "Geolocation Sensor"
 	},
+	"INPUT-DEVICE-CAPABILITIES": {
+		"href": "https://wicg.github.io/InputDeviceCapabilities/",
+		"title": "InputDeviceCapabilities"
```

Comment:
@RByers, WDYT about changing the title of the spec to have spaces? The only other spec name I'm aware of that is directly the name of the API is https://w3c.github.io/editing/execCommand.html

Reply:
I've sent https://github.com/WICG/InputDeviceCapabilities/pull/29 and will tentative change it here too.

Reply:
SGTM, thank you